pkg/parsing/impl/validator.go

```diff
@@ -220,8 +220,12 @@ type mutatingStmt struct {
 
 var _ parsing.MutatingStmt = (*mutatingStmt)(nil)
 
-func (s *mutatingStmt) GetQuery() (string, error) {
-	return s.node.String(), nil
+func (s *mutatingStmt) GetQuery(resolver parsing.WriteQueryResolver) (string, error) {
+	query, err := s.node.Resolve(resolver)
```

Comment:
>The go-sqlparser will define its own WriteQueryResolver interface, and we should use that one here instead of the one defined in go-validator.

Oh, forget about [this](https://github.com/tablelandnetwork/go-tableland/pull/424/files#r1057327426) comment.